test/adapters/postgres.test.js

```diff
@@ -87,7 +83,7 @@ if (process.env.PATIO_DB === "pg") {
         });
 
         it.should("correctly parse the schema", function () {
-            return comb.when(PG_DB.schema("test3"), PG_DB.schema("test4")).chain(function (schemas) {
+            return comb.when([PG_DB.schema("test3"), PG_DB.schema("test4")]).chain(function (schemas) {
```

Comment:
comb.when will eventually be replaced from a promise implementation such as - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all